### PR TITLE
Styles and legend

### PR DESCRIPTION
**Pull request by DingoEatingFuzz**
Looks like this now:

image

It's coming together :o

### CODE REVIEW

File: src/components/App/index.js [hunk 1]
```diff
@@ -54,6 +55,36 @@ const parameterGroupStyle = {
...
+  height: '100px',
+  transform: 'rotate(45deg) translate(-30%, -30%)',
+  margin: 'auto',
+  position: 'absolute',
+  zIndex: '1000',
+  left: '0',
+  right: '0',
+  border: '1px solid #ddd',
+};
+
```
Comment (nmuldavin) [inline src/components/App/index.js:87]: Nice work here, I probably wouldn't have done this at such a late hour!

File: src/components/MapLegend/index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+const cssCrossHatch = [
+  `repeating-linear-gradient(
+    45deg,
+    transparent,
+    transparent ${hatchWidth}px,
+    #FFF ${hatchWidth}px,
+    #FFF ${hatchWidth + strokeWidth}px,
+    transparent ${hatchWidth + strokeWidth}px,
+    transparent ${hatchWidth * 2}px
```
Comment (nmuldavin) [inline src/components/MapLegend/index.js:32]: Also nice work! Gotta look good on the imax ;)

**Review by nmuldavin — APPROVED** [review]
Looks good!

**Comment by JohnTasto** [comment]
Beautiful.

**Pull request merged**